views/dashboard/dashboard.hbs

```diff
@@ -40,7 +40,7 @@
             <div class="row lesson-cards">
                 <div>
                     {{#each events}}
-                    <a class="lesson-card" {{#if url}}href="{{url}}" {{/if}}{{#if alt}}title="{{alt}}" {{/if}}
+                    <a class="lesson-card" aria-label="{{title}}" {{#if url}}href="{{url}}" {{/if}}{{#if alt}}title="{{alt}}" {{/if}}
```

Comment:
Without an href a user won't be able to interact with the link. An empty href is valid html.

Reply:
Actually I take back my comment about empty href. This website suggests that we should not use them: https://webaim.org/techniques/keyboard/
![image](https://user-images.githubusercontent.com/25037067/88897774-1c28ba00-d24c-11ea-94d5-ba958ff1dcf0.png)
We can use something like href="#0" etc. instead.